docs/service-catalog/docs/001-overview-service-catalog.md

```diff
@@ -5,7 +5,7 @@ type: Overview
 
 The Service Catalog is a grouping of reusable, integrated services from all Service Brokers registered in Kyma. Its purpose is to provide an easy way for Kyma users to access services that the Service Brokers manage and use them in their applications.
 
-Due to the fact that Kyma runs on Kubernetes, you can easily run, in Kyma, a service that a third party provides and maintains, such as a database. Without extensive knowledge about the clustering of such a datastore service and the responsibility for its upgrades and maintenance, you can easily provision an instance of the software offering that a Service Broker registered in Kyma exposes, and bind it with an application running in the Kyma cluster.
+Due to the fact that Kyma runs on Kubernetes, you can easily instantiate a service instance, a service that a third party provides and maintains, such as a database and consume it from Kyma without extensive knowledge about the clustering of such a datastore service and the responsibility for its upgrades and maintenance, you can easily provision an instance of the software offering that a Service Broker registered in Kyma exposes, and bind it with an application running in the Kyma cluster.
```

Comment:
...``you can easily instantiate a service instance` ~, a service~ `that a third party provides and maintains, such as a database. You can also consume it from Kyma without extensive knowledge about the clustering of such a datastore service and the responsibility for its upgrades and maintenance. You can also easily provision an instance of the software offering that a Service Broker registered in Kyma exposes, and bind it with an application running in the Kyma cluster.``